source/ota.c

```diff
@@ -2365,6 +2365,10 @@ static OtaFileContext_t * parseJobDoc( const JsonDocParam_t * pJsonExpectedParam
     {
         err = OtaJobParseErrBadModelInitParams;
     }
+    else if( pFileContext->blocksRemaining > OTA_MAX_BLOCK_BITMAP_SIZE )
+    {
+        err = OtaJobParseErrBadModelInitParams;
```

Comment:
```suggestion
        LogWarn( ( "OTA size (%lu blocks) greater than can be tracked. Increase `OTA_MAX_BLOCK_BITMAP_SIZE`", pFileContext->blocksRemaining ) );
        err = OtaJobParseErrBadModelInitParams;
```

Let's add a warning message here so a resolution is clear to the user.

Reply:
Good call! I actually had wanted to do this but since the if statement before it didn't I thought it might be odd that we only add the error here. Now realizing that the `initDocModel()` function has its own logging.